Change "Provider" to "Source"

"Provider" is a confusing term, "source" is a more commonly used term that's easier to understand. Please replace all references to "provider" with "source".

I have replaced all references to "provider" with "source". It is building and serving fine. On running tests, all unit tests pass but e2e tests fail. Can somebody tell why is this happening?
This is the error it is throwing.
```
> node test/e2e/runner.js

internal/modules/cjs/loader.js:638
    throw err;
    ^

Error: Cannot find module '/home/cccatalog-frontend/test/e2e/runner.js'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Function.Module.runMain (internal/modules/cjs/loader.js:829:12)
    at startup (internal/bootstrap/node.js:283:19)
```

--- a/src/components/SearchGridFilter.vue
+++ b/src/components/SearchGridFilter.vue
@@ -39,7 +39,7 @@
           v-model="filter.provider"
           @input="onUpdateFilter"
           tag-placeholder="Add this as new tag"
-          placeholder="All Providers"
+          placeholder="All Sources"
           label="name"
           track-by="code"
           :options="providers"

--- a/src/pages/AboutPage.vue
+++ b/src/pages/AboutPage.vue
@@ -40,11 +40,11 @@
         Looking for the old CC Search portal? Visit
         <a href="https://oldsearch.creativecommons.org">https://oldsearch.creativecommons.org</a>.
       </p>
-      <h2>Providers</h2>
+      <h2>Sources</h2>
       <div class="page_provider-stats-ctr">
         <table class="page_provider-stats-table">
           <thead>
-            <th>Provider</th>
+            <th>Source</th>
             <th>Domain</th>
             <th># CC Licensed Works</th>
           </thead>

